content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -94,7 +94,8 @@ different Kubernetes components.
 | `ConfigurableFSGroupPolicy` | `false` | Alpha | 1.18 | 1.19 |
 | `ConfigurableFSGroupPolicy` | `true` | Beta | 1.20 | |
 | `CronJobControllerV2` | `false` | Alpha | 1.20 | 1.20 |
-| `CronJobControllerV2` | `true` | Beta | 1.21 | |
+| `CronJobControllerV2` | `true` | Beta | 1.21 | 1.21 |
+| `CronJobControllerV2` | `true` | GA | 1.22 | - |
```

Comment:
since this is now graduating, it needs to be moved below in the table [Feature gates for graduated or deprecated features](https://github.com/kubernetes/website/blob/a2eac5b39e7ae9adb961879552a9d76e7595f994/content/en/docs/reference/command-line-tools-reference/feature-gates.md#feature-gates-for-graduated-or-deprecated-features)

Reply:
sorry, just to be more succinct, all `CronJobControllerV2` entries from here now will go in that table.